packages/domainproperties/src/components/DomainForm.tsx

```diff
@@ -57,18 +66,22 @@ import {
 interface IDomainFormInput {
     domain: DomainDesign
     onChange: (newDomain: DomainDesign, dirty: boolean) => any
+    onToggle?: (collapsed: boolean, callback?: () => any) => any
     helpURL?: string
     helpNoun?: string
     showHeader?: boolean
     initCollapsed?: boolean
     collapsible?: boolean
-    markComplete?: boolean
+    controlledCollapse?: boolean
+    validate?: boolean
+    isNew?: boolean
+    panelStatus?: AssayPanelStatus
```

Comment:
Should the AssayPanelStatus const type be renamed since it isn't just for assay? Maybe just calling it PanelStatus? or DomainPanelStatus?